server/routes/api/controller/fund.js

```diff
@@ -0,0 +1,82 @@
+const express = require('express');
+const router = express.Router();
+const fundModel = require('../../../models/Fund');
+const auth = require('../../../middleware/auth');
+
+//Create Fund
+router.post('/create', auth, async (req, res) => {
+
+    const { name, description, goal, deadline } = req.body;
+    try {
+        const createFund = new fundModel({
+            user: req.user.id,
+            name,
+            description,
+            goal,
+            deadline
+        });
+        const created = await createFund.save();
+        res.send(created);
+    }
+    catch (err) {
+        console.log(err.message);
+        res.status(500).send('Serve Error');
+    }
+});
+
+//Update Fund
+router.put('/update', auth, async (req, res) => {
+    try {
+        //Finding the fund
+        let updateFund = await fundModel.findById({ user: req.user.id, _id: req.body.id });
+        if (!updateFund) {
+            return res.status(404).send('Fund not found');
+        }
+        //Update the found fund
+        const { id, name, description, goal, deadline } = req.body;
```

Comment:
we could possible move this ahead of line 31, so that we can reuse the `id` const var instead of reaching for `req.body.id` again. reduce repeated code :)